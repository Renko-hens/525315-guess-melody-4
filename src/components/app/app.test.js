import React from "react";
import renderer from "react-test-renderer";
import App from "./app.jsx";

describe(`Render Components from App`, () => {
  it(`Render App`, () =>{
    const tree = renderer.create(<App errorCount={3}/>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
