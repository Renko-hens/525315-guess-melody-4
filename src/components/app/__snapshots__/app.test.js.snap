// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Components from App Render App 1`] = `null`;
